examples/011_rst_workflow.py

```diff
@@ -0,0 +1,89 @@
+"""
+.. _rst_workflow_example:
+
+Failure Analysis of an MAPDL (RST) Model
+----------------------------------------
+
+The post-processing of a MAPDL (RST) model with layered elements which was not
+preprocessed by ACP is shown by this example. The difference between the RST
+only and ACP based workflow is that `composite` of :class:`.ContinuousFiberCompositesFiles`
```

Comment:
```suggestion
only and ACP-based workflow is that `composite` of the :class:`.ContinuousFiberCompositesFiles` class
```
Single backtick will only italicize the word. If this should be a code entity, use double backticks.